### PR TITLE
Create allow web traffic in ufw

### PR DESCRIPTION
**Pull request by csilva235**
Submitting document to assist customers in opening software firewall ports in UFW

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 1334 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-bd802b6d3/how-to/). valid for 14 days

**Comment by catlook** [comment]
@csilva235 final sign off, pls?

**Comment by csilva235** [comment]
Good to go! Thanks @catlook

**Comment by rackernexus** [comment]
Build 1450 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-036eda623/how-to/). valid for 14 days

File: content/cloud-servers/allow-web-traffic-in-ufw.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Prerequisites
+
+You need to have the following prerequisites:
+
+- Basic understanding of Secure Shell (SSH)
+
+- Sudo or administrative access to your server
+
+- A Cloud Server with Debian or Ubuntu&reg; running `ufw`
```
Comment (stephamon) [inline content/cloud-servers/allow-web-traffic-in-ufw.md]: "cloud server" shouldn't be capitalized

File: content/cloud-servers/allow-web-traffic-in-ufw.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+**IMPORTANT**: These steps are only for non-RackConnected Cloud servers. If you are using RackConnect, you can make similar changes on your dedicated firewall. 
+
+Log in to your server by using SSH and then complete the steps in the following sections for
+your preferred setup method.
+
+**NOTE**: While `ufw` is the default software firewall solution for most Debian-based distributions, you should note that `ufw` acts as a wrapper for `iptables` and you can disable the firewall. Before proceeding with the following steps, ensure that `ufw` is running on your server by running the following command:
+
+        systemctl status ufw
```
Comment (stephamon) [inline content/cloud-servers/allow-web-traffic-in-ufw.md]: This code needs to be part of the note block for it to format that way. You could bring it up to the previous line by writing "...ensure that `ufw` is running on your server by running `systemctl status ufw`." instead.

File: content/cloud-servers/allow-web-traffic-in-ufw.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+#### One-line command method
+
+Use the following steps for the one-line command method:
+
+1. Use the following one-line command to open the open the firewall ports:
```
Comment (stephamon) [inline content/cloud-servers/allow-web-traffic-in-ufw.md]: Delete line 36 and the number in 38 to keep this section simple.

**Comment by rackernexus** [comment]
Build 1460 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-155b6409d/how-to/). valid for 14 days

**Pull request merged**